python/lsst/daf/butler/queries/_expression_strings.py

```diff
@@ -239,8 +240,12 @@ def visitNumericLiteral(self, value: str, node: Node) -> _VisitorResult:
     def visitParens(self, expression: _VisitorResult, node: Node) -> _VisitorResult:
         return expression
 
-    def visitPointNode(self, ra: _VisitorResult, dec: _VisitorResult, node: Node) -> _VisitorResult:
-        raise NotImplementedError("POINT() function is not supported yet")
+    def visitPointNode(self, ra: _VisitorResult, dec: _VisitorResult, node: PointNode) -> _VisitorResult:
+        ra_value = _get_float_literal_value(ra, node.ra)
+        dec_value = _get_float_literal_value(dec, node.dec)
+
+        lon_lat = lsst.sphgeom.LonLat.fromDegrees(ra_value, dec_value)
```

Comment:
Did you notice whether those preexisting docs specifically said the values were supposed to be decimal degrees?  If we don't want to get into handling multiple angular units (to say nothing of `H:M:S` angles), decimal degrees is probably the best choice, but it definitely needs to be documented.

Reply:
The only thing it previously said about units was "similar to ADQL syntax", which implies [decimal degrees](https://www.ivoa.net/documents/ADQL/20180112/PR-ADQL-2.1-20180112.html#sec:functions.geom.point).

I had already tweaked the docs to say this:

> You can check for overlap of a region with a point using the ``POINT(ra, dec)`` syntax,
where ``ra`` and ``dec`` are specified as an ICRS sky position in degrees.

Is that a good way to phrase it?

Reply:
:+1: